grpc/nodejs/BUILD

```diff
@@ -63,6 +63,7 @@ ts_project(
         ":node_modules/@types/node",
         ":node_modules/typescript",
     ],
+    transpiler = "tsc",
```

Comment:
```
Traceback (most recent call last):
        File "/private/var/tmp/_bazel_spare/afea01e34d46fe314768544eee28004b/external/aspect_rules_ts/ts/private/ts_project.bzl", line 246, column 21, in _ts_project_impl
                fail(transpiler_selection_required)
Error in fail: 
######## Required Transpiler Selection ########

You must select a transpiler for ts_project rules, which produces the .js outputs.

Please read https://docs.aspect.build/rules/aspect_rules_ts/docs/transpiler

##########################################################
```

Here I use the transpiler that @alexjpwalker used for `platform`, not sure if anything more special is needed. As I understand, it might affect the runtime speed of the `ts` application if the transpiler is incorrect for this specific app, and I don't think that we want to compare them for our rules. Maybe @alexjpwalker could help us verifying it.

Reply:
Yes, as you have done, I'd rather use `tsc` here for simplicity. In my experience, TypeScript compilation usually completes in under 1 second, so performance is irrelevant.